docs/resources/as_group.md

```diff
@@ -189,6 +189,8 @@ The following arguments are supported:
 * `delete_instances` - (Optional, String) Whether to delete the instances in the AS group
     when deleting the AS group. The options are `yes` and `no`.
 
+* `enable` - (Optional, Bool) Whether to enable the AS Group. The options are `true` and `false`.
```

Comment:
we should add the description of default value (true).

Reply:
fixed